indico/modules/events/papers/models/papers.py

```diff
@@ -82,7 +82,8 @@ def can_comment(self, user):
     def can_submit(self, user):
         if not user:
             return False
-        return any(x.person.user == user for x in self.contribution.person_links if x.person.user)
+        return (any(x.person.user == user for x in self.contribution.person_links if x.person.user) or
```

Comment:
oh ..true :smile:  .. but there is also this check at the end ` if x.person.user` , I dont have that in the is_user_associated, I guess i should add it cause the user can be None right ?